docs/content/management.md

```diff
@@ -684,35 +684,37 @@ services:
         token: "bGFza2RqZmxha3NkamZsa2Fqc2Rsa2ZqYWtsc2RqZmtramRmYWxkc2tm"
 
 discovery:
-  name: /example/discovery
-  prefix: configuration
+  name: example
+  resource: /configuration/example/discovery.tar.gz
   service: acmecorp
 ```
 
 Using the boot configuration above, OPA will fetch discovery bundles from:
 
 ```
-https://example.com/control-plane-api/v1/configuration/example/discovery
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^
-services[discovery.service].url                          |                   |
-                                         + discovery.prefix  |
-                                                             + discovery.name
+https://example.com/control-plane-api/v1/configuration/example/discovery.tar.gz
```

Comment:
This isn't rendering as expected for me:
![Screenshot from 2019-09-26 10-51-35](https://user-images.githubusercontent.com/238885/65699168-afe99280-e04b-11e9-993e-9a0e8232c021.png)

Reply:
Check out the latest one, I updated it to match the bundles one earlier in the docs. Still not sure if its a *great* format but its at least better than before.